src/csharp/doc/docfx.json

```diff
@@ -0,0 +1,37 @@
+{
+  "metadata": [
+    {
+      "src": [
+        {
+          "files": ["Grpc.Core/Grpc.Core.csproj",
+                    "Grpc.Auth/Grpc.Auth.csproj",
+                    "Grpc.Core.Testing/Grpc.Core.Testing.csproj",
+                    "Grpc.HealthCheck/Grpc.HealthCheck.csproj",
+                    "Grpc.Reflection/Grpc.HealthCheck.csproj"],
+          "exclude": [ "**/bin/**", "**/obj/**" ],
+          "cwd": ".."
+        }
+      ],
+      "properties": { "TargetFramework": "net45" },
```

Comment:
One nit: does this not indicate only net45 support in the docs, and not netstandard1.5 support? (saw an example in https://dotnet.github.io/docfx/RELEASENOTE.html that used both a net46 and netstandard target, not sure if it's worth doing something similar).

Reply:
I think you can generate API docs for both if the APIs are different - in our case they aren't.
The generated docs doesn't contain any mention of net45 vs netstandard15, so leaving as is.